src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -2837,28 +2841,30 @@ def process_nodes(self, parent, nodes):
 
```

Comment:
Since we're here, line above should have `list[:py:class:`fparser.two.utils.Base`]`

Reply:
Done